### PR TITLE
mt5 fundtransfer usd to usd

### PR DESCRIPTION
**Pull request by ravali-deriv**
MT5 fund transfer, both deposit and withdrawal, USD to USD

card : https://app.clickup.com/t/20696747/CFDS-2826

### CODE REVIEW

**Comment by ravali-deriv** [comment]
This is successful execution of mt5 fundtransfer
Screenshot 2023-12-26 at 6 59 26 PM

File: cypress/e2e/wip/mt5/fundtransfer.cy.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  })
+  let transferAmount='10'
+  let mt5Account='Derived SVG'
+  let fiataccount='US Dollar'
+  function mt5fundtransfer(fromaccount,toaccount){
+    
+    cy.findByText("Transfer funds to your accounts", {
+        exact: true,
+      }).should("be.visible")
+    cy.get('[class="dc-dropdown-container account-transfer-form__drop-down"]').should("be.visible").click()
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/mt5/fundtransfer.cy.js:17]: lets not use class, use the testing library queries instead - https://testing-library.com/docs/cypress-testing-library/intro/. Please update all the locator to use testing library queries if possible.

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -186,6 +186,7 @@ Cypress.on('uncaught:exception', (err, runnable, promise) => {
 
 
 
+
```
Comment (meeyun-deriv) [inline cypress/support/e2e.js:189]: remove this changes as this is not needed

File: cypress/e2e/wip/mt5/fundtransfer.cy.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+import { waitFor } from '@testing-library/dom'
+
+describe('QATEST-5778, QATEST-5781, QATEST-5615', () => {
+  beforeEach(() => {
+    cy.c_login()
+    cy.c_visitResponsive('/appstore/traders-hub', 'large')
+  })
+  let transferAmount='10'
+  let mt5Account='Derived SVG'
+  let fiataccount='US Dollar'
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/mt5/fundtransfer.cy.js:11]: standardize the naming convention, it should be `fiatAccount`. Please check across the entire PR on the naming convention.

Same for file name too, please update `fundtransfer.cy.js` to `fundTransfer.cy.js`

File: cypress/e2e/wip/mt5/tradersHub.cy.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import '@testing-library/cypress/add-commands'
+
+describe('QATEST-5778, QATEST-5781, QATEST-5615', () => {
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/mt5/tradersHub.cy.js:3]: where is QATEST-5778 and QATEST-5781 coming from? I cant seem to find them in Clickup

File: cypress/e2e/wip/mt5/fundtransfer.cy.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
+import '@testing-library/cypress/add-commands'
+import { waitFor } from '@testing-library/dom'
+
+describe('QATEST-5778, QATEST-5781, QATEST-5615', () => {
+  beforeEach(() => {
+    cy.c_login()
+    cy.c_visitResponsive('/appstore/traders-hub', 'large')
```
Comment (meeyun-deriv) [inline cypress/e2e/wip/mt5/fundtransfer.cy.js:7]: why do we handle the desktop view and not responsive first?

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/mt5/fundtransfer.cy.js (hunk 3 above)
Comment (ravali-deriv) [reply cypress/e2e/wip/mt5/fundtransfer.cy.js:11]: This is done

File: cypress/e2e/wip/mt5/tradersHub.cy.js (hunk 4 above)
Comment (ravali-deriv) [reply cypress/e2e/wip/mt5/tradersHub.cy.js:3]: I have changed this to correct description

**Review by markodonnell-deriv — APPROVED** [review]

**Comment by ravali-deriv** [comment]
closing this PR because I have created a new PR for this task 
new PR - https://github.com/deriv-com/e2e-deriv-app/pull/44
card - https://app.clickup.com/t/20696747/CFDS-2826

**Pull request closed without merging**